lib/stream/followed_tag.rb

```diff
@@ -16,7 +16,12 @@ def title
 
   # @return [ActiveRecord::Association<Post>] AR association of posts
   def posts
-    @posts ||= StatusMessage.user_tag_stream(user, tag_ids)
+    if tag_ids.empty?
+      @posts = StatusMessage.none
+    else
+      @posts ||= StatusMessage.any_user_tag_stream(user, tag_ids)
+    end
```

Comment:
I'd rather move this condition under `any_user_tag_stream`. We probably want such behavior in any usage of `any_user_tag_stream`